src/components/__tests__/setupTests.js

```diff
@@ -2,4 +2,56 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import { RecoilRoot } from 'recoil'
+import userEvent from '@testing-library/user-event'
+
+//components
+import SearchBox from '../SearchBox/SearchBox'
+
+test('[SearchBox] sanity check, searchbox component renders without errors', async () => {
+  render(
+    <RecoilRoot>
+      <SearchBox placeholder='test name' />
+    </RecoilRoot>
+  )
+})
+
+test('[SearchBox] takes custom placeholder prop', async () => {
+  render(
+    <RecoilRoot>
+      <SearchBox placeholder='test name' />
+    </RecoilRoot>
+  )
+
+  const testPlaceholder = await screen.findByPlaceholderText(/test NAME/i)
+  expect(testPlaceholder).toBeInTheDocument()
+})
+
+test('[SearchBox] aria label exists', async () => {
+  render(
+    <RecoilRoot>
+      <SearchBox placeholder='test name' />
+    </RecoilRoot>
+  )
+
+  const ariaLabel = await screen.findByLabelText(/Search Box Text Field/i)
+  expect(ariaLabel).toBeInTheDocument()
+})
+
+test('[SearchBox] submitted searchbox text renders on {enter}', async () => {
```

Comment:
I really like this test! It looks like a great way to actually make sure the recoil atom is receiving information

Reply:
Many thanks!